rust/tests/namespace.ts

```diff
@@ -2,10 +2,7 @@ import * as anchor from "@project-serum/anchor";
 import { BN } from "@project-serum/anchor";
 import * as splToken from "../node_modules/@solana/spl-token";
 import * as mpl from "@metaplex-foundation/mpl-token-metadata";
-import {
-  Namespace,
-  IDL as NamespaceProgramIDL,
-} from "../target/types/namespace";
+import { IDL as NamespaceProgramIDL } from "../target/types/namespace";
```

Comment:
Are you planning to update this to point to the idl in the js lib? https://github.com/raindrops-protocol/raindrops/pull/32